lib/instance.js

```diff
@@ -554,6 +554,14 @@ Instance.prototype.save = function(options) {
     }
   }
 
+  if (updatedAtAttr && !options.silent && options.fields.indexOf(updatedAtAttr) !== -1) {
+    this.dataValues[updatedAtAttr] = this.Model.$getDefaultTimestamp(updatedAtAttr) || Utils.now(this.sequelize.options.dialect);
+  }
+
+  if (this.isNewRecord && createdAtAttr && !this.dataValues[createdAtAttr]) {
+    this.dataValues[createdAtAttr] = this.Model.$getDefaultTimestamp(createdAtAttr) || Utils.now(this.sequelize.options.dialect);
+  }
```

Comment:
`now` should still be a valuable to ensure there aren't any possible ms differences.